docs/guides/lint-asyncapi.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
Q -- will the linebreaks I'm seeing in the markdown resolve to the published docs? (just checking in case it's a copy / paste formatting issue or something similar)
 
![Screenshot 2023-09-08 at 4 14 41 PM](https://github.com/Redocly/redocly-cli/assets/20376095/d21be0c8-9776-49cf-b718-b4a6768c47e3)

Reply:
No, markdown only renders new paragraphs after a blank line. I softwrapped this text so that it's easier to see the diff when things change (we don't have a standard approach for this yet)

Reply:
We do have a standard approach: https://sembr.org/